addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -692,30 +692,29 @@ function factory(dependencies) {
         _notifyNewChannelMessageWhileOutOfFocus({ channel, message }) {
             const author = message.author;
             const messaging = this.env.messaging;
-            let notificationTitle;
+            let notificationHeader;
             if (!author) {
-                notificationTitle = this.env._t("New message");
+                notificationHeader = this.env._t("New message");
             } else {
                 const authorName = author.nameOrDisplayName;
```

Comment:
You can escape only once here instead of on every condition.
```suggestion
                const escapedAuthorName = owl.utils.escape(author.nameOrDisplayName);
```